codegen/java-gen/src/test/resources/tests/action-service/named-new-style/generated-unmanaged/org/example/service/MyServiceNamedAction.java

```diff
@@ -6,6 +6,7 @@
 import com.google.protobuf.Empty;
 
 // This class was initially generated based on the .proto definition by Akka Serverless tooling.
+// This is the implementation for the Action Service described in your com/example/different/example-action.proto file.
```

Comment:
Here's the proof that it works as expected.

Reply:
the package defined in the service is `com.example.service` while the `.proto` is in `com/example/different`.

Reply:
Nice!